src/NewsService.Data.Provider.MsSql.Ef/Migrations/20210812164100_AddServicePropertiesToDb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using LT.DigitalOffice.NewsService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(NewsServiceDbContext))]
+    [Migration("20210812164100_AddServicePropertiesToDb")]
+    public class AddServicePropertiesToDb : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: nameof(DbNews.CreatedBy),
+                table: "News",
+                nullable: false);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: nameof(DbNews.CreatedAtUtc),
+                table: "News",
+                nullable: false);
+
+            migrationBuilder.AddColumn<Guid>(
+                name: nameof(DbNews.ModifiedBy),
+                table: "News",
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: nameof(DbNews.ModifiedAtUtc),
+                table: "News",
+                nullable: true);
+
+            migrationBuilder.DropColumn(
```

Comment:
you could just rename this column

Reply:
???

Reply:
Egor sad, what you can use migrationBuilder.RenameColumn... rename CreatedAt to CreatedAtUtc.

Reply:
Egor isn't sad=), I said

Reply:
I think you can leave what you have, I just said as differently as possible